liferay-echo/liferay-portal
# LPS-195619 Convert old AUI in staging-version using JS

## Motivation

[Link to ticket](https://liferay.atlassian.net/browse/LPS-1956198)

## Test Scenario 1

1. Activate staging (with page versioning)
2. Create a new Widget Page.
3. Check the kebab menu actions.
4. Check the behaviour for the toggle of “Ready for Publish Process”    

https://github.com/liferay-echo/liferay-portal/assets/93203423/5874c316-5494-4bd2-bc4f-9e26cd3a90dd

5. Go to Workflow. Look for “page” and activate Page Revision.
6. Navigate again to the widget page and check the behaviour for the button of “Submit for Workflow”

https://github.com/liferay-echo/liferay-portal/assets/93203423/eca6314f-4ed3-4dec-bdef-438df1b05dff
